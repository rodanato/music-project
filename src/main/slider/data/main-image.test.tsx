

import React from 'react';
import renderer from 'react-test-renderer';
import MainImageAtom from './main-image.atom';

it('MainImageAtom snapshot example', () => {
  const tree = renderer.create(
    <MainImageAtom url="https://dummyimage.com/111x92/000000/fff"/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});