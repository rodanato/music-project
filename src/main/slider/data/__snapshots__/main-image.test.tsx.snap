// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainImageAtom snapshot example 1`] = `
<img
  alt=""
  className="css-bbf2li-MainImageAtom"
  src="https://dummyimage.com/111x92/000000/fff"
/>
`;
